code/modules/religion/religion_sects.dm

```diff
@@ -213,7 +223,141 @@
+	You earn favor by getting others to join the crusade, and you may spend favor to announce a battle, bypassing some conditions to attack."
+
+/datum/religion_sect/honorbound/on_conversion(mob/living/carbon/new_convert)
+	if(!ishuman(new_convert))
+		to_chat("<span class='warning'>[GLOB.deity] has no respect for lower creatures, and refuses to make you honorbound.</span>")
+	else
+		var/datum/dna/holy_dna = new_convert.dna
+		holy_dna.add_mutation(/datum/mutation/human/honorbound)
+
+/datum/religion_sect/burden
+	name = "Punished God"
+	desc = "A sect that desires to feel the pain of their god."
+	altar_icon_state = "convertaltar-burden"
+	alignment = ALIGNMENT_NEUT
+	convert_opener = "\"To feel the freedom, you must first understand captivity.\"<br>\
+	Incapacitate yourself in any way possible. Bad mutations, lost limbs, traumas, even addictions. You will learn the secrets of the universe \
+	from your defeated shell."
+	//a list for keeping track of how burdened each member is
+
+/datum/religion_sect/burden/on_conversion(mob/living/carbon/new_convert)
+	if(!ishuman(new_convert))
+		to_chat("<span class='warning'>[GLOB.deity] needs higher level creatures to fully comprehend the suffering. You are not burdened.</span>")
+	else
+		var/datum/dna/holy_dna = new_convert.dna
+		holy_dna.add_mutation(/datum/mutation/human/burdened)
+
+/datum/religion_sect/burden/tool_examine(mob/living/carbon/human/burdened) //display money policy
+	var/datum/mutation/human/burdened/burdenmut = burdened.dna.check_mutation(/datum/mutation/human/burdened)
+	if(burdenmut)
+		return "<span class='notice'>You are at burden level [burdenmut.burden_level]/6.</span>"
+	return "<span class='notice'>You are not burdened.</span>"
+
+#define MINIMUM_YUCK_REQUIRED 5
+
+/datum/religion_sect/maintenance
+	name = "Maintenance God"
+	desc = "A sect based around the maintenance shafts of the station."
+	altar_icon_state = "convertaltar-maint"
+	alignment = ALIGNMENT_EVIL //while maint is more neutral in my eyes, the flavor of it kinda pertains to rotting and becoming corrupted by the maints
+	convert_opener = "\"Your kingdom in the darkness.\"<br>\
+	Sacrifice the organic slurry created from rats dipped in welding fuel to gain favor. Exchange favor to adapt to the maintenance shafts."
+	rites_list = list(/datum/religion_rites/maint_adaptation, /datum/religion_rites/adapted_food, /datum/religion_rites/ritual_totem)
+	desired_items = list(/obj/item/reagent_containers)
+
+/datum/religion_sect/maintenance/sect_bless(mob/living/blessed_living, mob/living/user)
+	if(!ishuman(blessed_living))
+		return TRUE
+	var/mob/living/carbon/human/blessed = blessed_living
+	if(blessed.reagents.has_reagent(/datum/reagent/drug/maint/sludge))
+		to_chat(blessed, "<span class='warning'>[GLOB.deity] has already empowered them.</span>")
+		return TRUE
+	blessed.reagents.add_reagent(/datum/reagent/drug/maint/sludge, 5)
```

Comment:
note that maint sludge does deal a little bit of tox damage over time

how many times do you have to bible smack someone (and wait for periods of time in-between smacks for the sludge to run its course) before they'll end up addicted to sludge?

Reply:
Addictions are really dumb right now so you'll pretty much never get addicted. You'll be dead from toxins way way faster